packages/console/src/hooks/use-api.ts

```diff
@@ -157,7 +157,9 @@ export const useStaticApi = ({
                 isCloud &&
                 isDevFeaturesEnabled &&
                 isAuthenticated &&
-                ['POST', 'PUT', 'DELETE'].includes(request.method) &&
+                (['POST', 'PUT', 'DELETE'].includes(request.method) ||
+                  // Special case for `/sign-in-exp` endpoint (MFA), which is a PATCH request.
+                  (request.method === 'PATCH' && request.url.endsWith('/sign-in-exp'))) &&
```

Comment:
I think special case should be handled in its own page, not in the `use-api` hook